eng/Sources.props

```diff
@@ -3,6 +3,13 @@
 
   <PropertyGroup Label="RestoreSources">
     <RestoreSources>$(DotNetRestoreSources)</RestoreSources>
+
+    <!-- This feed contains the 2.2.0 RTM runtime. This can be removed after it is released to nuget.org. -->
```

Comment:
Should we add a post-release activity to clean up things like this?

Reply:
I don't think this is necessary.  The dotnet-core blob and myget feeds contain the 2.2.0 packages

Reply:
Yeah, I'll open a bug on aspnetcore-internal to track this.

Reply:
> The dotnet-core blob and myget feeds contain the 2.2.0 packages

Cool, i'll remove this feed now. Didn't see this comment before I merged.